Sources/DolbyIORTSCore/State/StateMachine.swift

```diff
@@ -111,35 +90,26 @@ final actor StateMachine {
 
     func onSubscribed() {
         switch currentState {
-        case .subscribing:
-            currentState = .subscribed(.init())
+        case .subscribed:
+            // No-op
+            break
```

Comment:
Do we receive `onSubscribed` event multiple times for one subscribe call ?

Reply:
no, only once. But sometimes it comes just after onVideoTrack / onAudioTrack

Reply:
If it's after the video and audio tracks that arrive before the onActive call - I think they can be ignored.

Reply:
seems like it only happens with single stream.. 
Thanks, reverted the change.